### ISSUE TITLE
Inconsistent test: test_backend_close_on_user_revoke

### ISSUE THREAD

**Issue by vxgmichel** [Scille/parsec-cloud#807]
```
================================== FAILURES ===================================
______________________ test_backend_close_on_user_revoke ______________________
[gw0] win32 -- Python 3.6.8 c:\python36-x64\python.exe
value = <trio.Nursery object at 0x000000E4190987F0>
    async def yield_(value=None):
>       return await _yield_(value)
c:\python36-x64\lib\site-packages\async_generator\_impl.py:106: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
c:\python36-x64\lib\site-packages\async_generator\_impl.py:99: in _yield_
    return (yield _wrap(value))
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
backend = <parsec.backend.app.BackendApp object at 0x000000E419098D68>
alice_backend_sock = <tests.common.FreezeTestOnTransportError object at 0x000000E4190A54A8>
backend_sock_factory = <function backend_sock_factory.<locals>._backend_sock_factory at 0x000000E418E41730>
bob = LocalDevice(organization_addr=BackendOrganizationAddr(url=parsec://bob.dev1.example.com:9999/CoolOrg?no_ssl=true&rvk=M...st_key=<parsec.crypto.SecretKey object at 0xe41851b6a8>, local_symkey=<parsec.crypto.SecretKey object at 0xe41851bc48>)
bob_revocation_from_alice = b'\xcbl\xb4\xb5\xe7EM~<\xc1x\xe1\xbbxXz\x8d\x94C5\xe4\x16\xeb?\x8d\xb3\x89\xc5\x1b\xf45"\x82\x84\xd0\x81\x1dc\xe6\x85+...%\x99\xb9\xa9\xc5%\x89\xb9\x05\xd7\x19\x1d\xaf\x97\xfc\xfeXf\xd8\xbc\xa4\xa4\xb2 uGQjY~vjJ<X{rjQIfZfrbI*\x00\xe8e$\x83'
    @pytest.mark.trio
    async def test_backend_close_on_user_revoke(
        backend, alice_backend_sock, backend_sock_factory, bob, bob_revocation_from_alice
    ):
        async with backend_sock_factory(
            backend, bob, freeze_on_transport_error=False
        ) as bob_backend_sock:
            rep = await user_revoke(
                alice_backend_sock, revoked_user_certificate=bob_revocation_from_alice
            )
            assert rep["status"] == "ok"
            # bob cannot send new command
            with pytest.raises(TransportError):
>               await bob_backend_sock.send(packb({"cmd": "ping", "ping": "foo"}))
E               Failed: DID NOT RAISE <class 'parsec.api.transport.TransportError'>
..\tests\backend\user\test_user_revoke.py:50: Failed
```

**Comment by touilleMan** [comment]
fixed by #814

**Comment by vxgmichel** [comment]
Seen again: https://ci.appveyor.com/project/touilleMan/parsec-cloud/builds/29043635

**Comment by touilleMan** [comment]
![image](https://user-images.githubusercontent.com/3187637/69429769-9fe0ed00-0d34-11ea-9f14-5f94a41484f3.png)

**Comment by vxgmichel** [comment]
And again: https://ci.appveyor.com/project/touilleMan/parsec-cloud/builds/29137210

**Comment by vxgmichel** [comment]
And again: https://ci.appveyor.com/project/touilleMan/parsec-cloud/builds/29156463

**Comment by vxgmichel** [comment]
And again: https://ci.appveyor.com/project/touilleMan/parsec-cloud/builds/29181125?fullLog=true

**Comment by vxgmichel** [comment]
Potentially fixed by #941.
